### ISSUE TITLE
Detect Octane Flavor

### ISSUE THREAD

**Issue by KTanAug21** [fly-apps/dockerfile-laravel#31]
**What:**
https://flyio.slack.com/archives/C042W39VAMB/p1710787888107569
Detect flavors of octane: 

> 1. FrankenPHP
    > Presence of frankenphp file in base directory
> 2. RoadRunner
    > .rr dir in base directory
    > keywords in composer.json
> 3. Swoole
    > This is actually a default option in the dockerfile generatedby flyctl for laravel apps with octane
maybe 
    >  Can this be detectable if the [swoole extension](https://openswoole.com/docs/get-started/installation#enable-open-swoole-extension-in-php) is available in the current dev environment? So check php.ini? 

**Why:**
https://flyio.slack.com/archives/C042W39VAMB/p1711553910059719?thread_ts=1711549635.827029&cid=C042W39VAMB

TLDR: detectables should be defaultables.

**Comment by KTanAug21** [comment]
Closed by #29
